src/components/AuditLogsTable/AuditLogsTable.tsx

```diff
@@ -122,20 +153,43 @@ const AuditLogsTable = ({ showModel = false }: Props) => {
 
   return (
     <div className="audit-logs__table">
-      <Button
-        onClick={() => {
-          fetchAuditEvents();
-          setQueryParams({ page: "1" });
-        }}
-      >
-        Refresh
-      </Button>
+      <div className="audit-logs__filter">
+        <Button
+          onClick={() => {
+            setQueryParams({ page: "1" });
+            fetchAuditEvents(limit);
+          }}
+        >
+          Refresh
+        </Button>
+        <div className="audit-logs__pagination">
+          <Pagination
+            onForward={() => {
+              setQueryParams({ page: `${Number(page) + 1}` });
```

Comment:
If you like the suggestion above about converting to a number then this could be `page: (page+1).toString()`